content/rc/administration/setup/customize-flexible-plan.md

```diff
@@ -0,0 +1,105 @@
+---
+Title: Customize a Flexible plan
+description:
+weight: 30
+alwaysopen: false
+categories: ["RC"]
+aliases: /rc/administration/customize-pro/
+---
+
+When you create a subscription with a Flexible plan, you customize it to your business needs.
```

Comment:
Where in the docs do we define "Flexible plan". Can we link to that from here?

Reply:
Fixed